Packs/ExpanseV2/Integrations/ExpanseV2/ExpanseV2.py

```diff
@@ -262,6 +314,29 @@ def list_providers(self, limit: int = DEFAULT_RESULTS) -> Iterator[Any]:
             params=params
         )
 
+    def list_pocs(self, limit: int = DEFAULT_RESULTS) -> Iterator[Any]:
+        params = dict(limit=limit)
+        return self._paginate(
+            method='GET',
+            url_suffix='/v2/annotation/point-of-contact',
+            params=params
+        )
+
+    def create_poc(self, email: str, first_name: Optional[str], last_name: Optional[str], role: Optional[str],
+                   phone: Optional[str]) -> Dict[str, Any]:
+        data: Dict = {
+            'email': email,
+            'firstName': first_name,
+            'lastName': last_name,
+            'phone': phone,
+            'role': role
```

Comment:
This will become a `null` in the payload, which the API should be able to handle. Also this function is only called by one other function which it looks like should always force an empty value to be an empty string, so I think this should be safe.